source/Makefile.vars

```diff
@@ -1,42 +1,21 @@
 FORTRAN       = ifort
-
-CPLUSPLUS      = icpc
-#CPLUSPLUS     = /public/intel2017/bin/icpc
-
+CPLUSPLUS     = icpc
 CPLUSPLUS_MPI = mpiicpc
-#CPLUSPLUS_MPI = /public/intel2017/impi/2017.1.132/intel64/bin/mpiicpc
 
-#==========================
-# LIB and INCLUDE
-#==========================
-HONG_LAPACK     = -D__MKL -DMKL_ILP64
-LAPACK_DIR    	= $(MKLROOT)
-LAPACK_INCLUDE  = -I${LAPACK_DIR}/include
-LAPACK_LIB_DIR  = ${LAPACK_DIR}/lib/intel64
-LAPACK_LIB      = -L${LAPACK_LIB_DIR} -Wl,--start-group -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -lmkl_scalapack_lp64 -lmkl_blacs_intelmpi_lp64 -Wl,--end-group -Wl,-rpath=${LAPACK_LIB_DIR}
+LAPACK_DIR    = $(MKLROOT)
+# LAPACK_DIR   = /public/software/parallel_studio_xe_2019_update3/compilers_and_libraries_2019/linux/mkl
 
-HONG_FFTW       = -D__FFTW3
-FFTW_DIR		= /home/liyuanbo/soft/fftw
-FFTW_INCLUDE    = -I${FFTW_DIR}/include
-FFTW_LIB_DIR    = ${FFTW_DIR}/lib
-FFTW_LIB        = -L${FFTW_LIB_DIR} -lfftw3 -Wl,-rpath=${FFTW_LIB_DIR}
+FFTW_DIR      = /public/software/fftw-3.3.8
 
-ELPA_DIR		= /home/liyuanbo/soft/elpa-2021.05.002/build
-ELPA_INCLUDE    = -I${ELPA_DIR}/include/elpa-2021.05.002
-ELPA_LIB_DIR	= ${ELPA_DIR}/lib
-ELPA_LIB     	= -L${ELPA_LIB_DIR} -lelpa -Wl,-rpath=${ELPA_LIB_DIR}
+ELPA_DIR      = /public/software/elpa-gitlab
 
-#LIBXC_INCLUDE   = -I${LIBXC_DIR}/include
-#LIBXC_LIB_DIR   = ${LIBXC_DIR}/lib
-#LIBXC_LIB       = -L${LIBXC_LIB_DIR} -lxc -Wl,-rpath=${LIBXC_LIB_DIR}
+CEREAL_DIR    = /public/software/cereal-1.3.2
 
-CEREAL_DIR 		= /home/liyuanbo/soft/cereal/
-CEREAL_INCLUDE 	= -I${CEREAL_DIR}/include
+# LIBXC_DIR     = /public/software/libxc-5.0.0
 
-#============DeePKS================
-#LIBTORCH_DIR = /home/fortneu49/soft/libtorch
-#LIBNPY_DIR = /home/fortneu49/soft/libnpy/include
+# LIBTORCH_DIR  = /public/software/libtorch
+# LIBNPY_DIR    = /public/software/libnpy
 
 OBJ_DIR = obj
-OBJ_DIR_serial = obj
-NP      = 14
+OBJ_DIR_serial = obj_serial
```

Comment:
@dyzheng Any update on this change?

Reply:
I don't know detail about Makefile updates, but linpz suggested that the update in this PR would benefit for users who are willing to use Makefile for compiling.